src/System.IO.Ports/tests/Legacy/SerialPort/DosDevices.cs

```diff
@@ -16,27 +17,28 @@ public DosDevices()
         Initialize();
     }
 
+/*
```

Comment:
Yeah, just dump it- we have history here if we need it. I'm going to merge in the current state as it is currently green- you can throw this in another PR.